lib/pharos/kube.rb

```diff
@@ -56,15 +58,15 @@ def self.config_exists?(host)
     # Shortcuts / compatibility:
 
     def self.apply_stack(host, name, vars = {})
-      session(host).stack(name, vars).apply
+      session(host).stack(name, RESOURCE_PATH, vars).apply
```

Comment:
Needs to be`File.join(RESOURCE_PATH, name)`?

Reply:
Yes.

Reply:
Fixed.